iquip/apps/builder.py

```diff
@@ -587,3 +604,33 @@ def onSubmitted(self, rid: int):
     def frames(self) -> Tuple[Tuple[str, BuilderFrame]]:
         """Overridden."""
         return (("", self.builderFrame),)
+
+class _ScanEntry(QWidget):
+    """Entry class for a Scannable argument.
+    
+    Attributes:
+        name: The name of the experiment.
+    """
+    def __init__(
+        self,
+        name: str,
+        argInfo: Dict[str, Any],
+        parent: Optional[QWidget] = None
+        ):
+        """Extended.
+
+        Args:
+            name: The name of the experiment.
```

Comment:
You have already written this explanation in the attribute section, hence please replace it with "name: See the attributes section.".